src/i18n/es.js

```diff
@@ -38,11 +38,11 @@ const messages_es = {
   agencies: {
     breadcrumb: 'Planes',
     breadcrumb_url: `${urlBuyMonthly}`,
-    feature_1: 'Acceso al panel de Client Manager para gestionar múltiples cuentas.',
-    feature_2: 'Administración de perfiles de usuario y permisos de acceso para tus clientes.',
-    feature_3: 'Uso ilimitado de las funcionalidades de Doppler.',
-    feature_4: 'Reportes avanzados sobre el desempeño de las Campañas.',
-    feature_5: 'Asesoramiento exclusivo para gestionar las cuentas de tus clientes.',
+    feature_access: 'Acceso al panel de Client Manager para gestionar las cuentas de tus clientes.',
+    feature_admin: 'Administración de perfiles de usuario y permisos de acceso para tus clientes.',
+    feature_consultancy: 'Asesoramiento exclusivo para gestionar múltiples cuentas.',
```

Comment:
Here in spanish we have `Asesoramiento exclusivo` but in english is capitalized `Exclusive Advisory` is it ok @jmonacchi ?

Reply:
I think it should be either `Asesoramiento Exclusivo` and `Exclusive Advisory` or both without capitals.

Reply:
@cbernat @silsanchez8 In english it should be Exclusive advisory.